src/Sylius/Component/Core/spec/Factory/CartItemFactorySpec.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is a part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Core\Factory;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Factory\CartItemFactory;
+use Sylius\Component\Core\Factory\CartItemFactoryInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Variation\Resolver\VariantResolverInterface;
+
+/**
+ * @mixin CartItemFactory
+ *
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+final class CartItemFactorySpec extends ObjectBehavior
+{
+    function let(FactoryInterface $decoratedFactory, VariantResolverInterface $variantResolver)
+    {
+        $this->beConstructedWith($decoratedFactory, $variantResolver);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Component\Core\Factory\CartItemFactory');
```

Comment:
For now, we've been using fully qualified class name as a convention, but [since PHPSpec 3.0](https://github.com/phpspec/phpspec/releases/tag/3.0.0) default implementation of ``it_is_initializable`` spec is using ``::class``, so we could think about changing it ;)